pydoctor/driver.py

```diff
@@ -32,9 +34,18 @@ def findClassFromDottedName(dottedname, optionname):
 
 MAKE_HTML_DEFAULT = object()
 
+def check_path(option, opt, value):
+    try:
```

Comment:
it would be nice to have a docstring explining what is this checking .

the check_path is a generic name in the context of pydoctor.driver

Reply:
I'll add one.